### PR TITLE
Add AppVeyor config to build wheels for Windows

### PR DESCRIPTION
**Pull request by andrewleech**
Add a basic configuration for building wheels for windows using AppVeyor.

Still requires configuration in the appropriate AppVeyor account and wheelhouse credentials encrypted and added to file.

For more info see:
https://github.com/scikit-image/scikit-image/issues/2514

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "python -m pip install -U pip"
+
+  # Check that we have the expected version and architecture for Python
+  - "python --version"
+  - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+  - "pip --version"
+
+  # Get stdint headers needed by tifffile.c.
+  - "curl https://raw.githubusercontent.com/chemeris/msinttypes/master/inttypes.h -o skimage/external/tifffile/inttypes.h"
```
Comment (matthew-brett) [inline .appveyor.yml]: The scikit-image repository is not at the root of this repo, but in a directory called`scikit-image`...

Comment (andrewleech) [reply .appveyor.yml]: Yes, line 40 cd's to this directory before operating on the repo.

**Comment by matthew-brett** [comment]
@andrewleech - I've refactored on top of this PR over at #6 - would you mind taking a look?   If #6 is OK, is it OK to close this one?

**Comment by andrewleech** [comment]
Absolutely. I never had the means to actually test this work, it really could only ever be a starting point. 

Thanks for taking it on!

**Pull request closed without merging**